pkg/agent/endpoints/workload/handler.go

```diff
@@ -58,7 +58,7 @@ func (h *Handler) FetchX509SVID(_ *workload.X509SVIDRequest, stream workload.Spi
 
 	selectors := attestor.New(&config).Attest(pid)
 
-	subscriber := h.Manager.Subscribe(selectors)
+	subscriber := h.Manager.NewSubscriber(selectors)
```

Comment:
I am not sure which one of these is considered idiomatic go... maybe @azdagron has an opinion here? /me prefers the verb since it's shorter. I guess I don't care _too_ much, but it would be good to get some general agreement so we are all following the same conventions

Reply:
maybe something like the following?
```
SubscribeToCacheChanges(key cache.Selectors) cache.Subscriber
SubscribeToSVIDChanges() observer.Stream
SubscribeToBundleChanges() observer.Stream
```

Reply:
Makes sense to me, I will update them.

Reply:
Thinking it better, I will use SubscribeToCacheChanges :slightly_smiling_face: